cirq-google/cirq_google/workflow/processor_record.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import abc
+import dataclasses
+
+import cirq
+import cirq_google as cg
+from cirq._compat import dataclass_repr
+from cirq_google.engine.virtual_engine_factory import (
+    MOST_RECENT_TEMPLATES,
+    _create_device_spec_from_template,
```

Comment:
Should this function not have a `_` prefix if it's imported here?

Reply:
My impression is that `_` implies it's private within a module, but I'm not a Python guru so not a strong preference :) According to [The naming guide from "PEP 8 -- Style Guide for Python Code"](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles):

"`_single_leading_underscore`: weak "internal use" indicator. E.g. from M import * does not import objects whose names start with an underscore."

This seems to imply that objects used outside a module shouldn't have an underscore.